R/bpmnVisualization.R

```diff
@@ -59,13 +59,13 @@ display <- function(bpmnXML, width = NULL, height = NULL, elementId = NULL) {
 #' @name bpmnVisualization-shiny
 #'
 #' @export
-bpmnVisualizationOutput <- function(outputId, width = '100%', height = '400px') {
+bpmn_visualization_output <- function(outputId, width = '100%', height = '400px') {
   htmlwidgets::shinyWidgetOutput(outputId, 'bpmnVisualization', width, height, package = 'bpmnVisualization')
 }
 
 #' @rdname bpmnVisualization-shiny
 #' @export
-renderBpmnVisualization <- function(expr, env = parent.frame(), quoted = FALSE) {
+render_bpmn_visualization <- function(expr, env = parent.frame(), quoted = FALSE) {
```

Comment:
⚠️ htmlWidgets expects a function named `render<upper_case_package_name>` for the R-binding: http://www.htmlwidgets.org/develop_intro.html#r-binding

Reply:
done in f9fe264